tests/clv/test_plotting.py

```diff
@@ -0,0 +1,52 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import pytest
+
+from pymc_marketing.clv.plotting import (
+    plot_customer_exposure,
+    plot_frequency_recency_matrix,
+    plot_probability_alive_matrix,
+)
+
+
+@pytest.fixture
+def test_summary_data() -> pd.DataFrame:
+    return pd.read_csv("tests/clv/datasets/test_summary_data.csv", index_col=0)
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"colors": ["blue", "red"]},
+        {"labels": ["Customer Recency", "Customer T"]},
+    ],
+)
+def test_plot_customer_exposure(test_summary_data, kwargs) -> None:
+    ax: plt.Axes = plot_customer_exposure(test_summary_data, **kwargs)
+
+    assert isinstance(ax, plt.Axes)
+
+
+def test_plot_cumstomer_exposure_with_ax(test_summary_data) -> None:
+    ax = plt.subplot()
+    plot_customer_exposure(test_summary_data, ax=ax)
+
+    assert isinstance(ax, plt.Axes)
```

Comment:
Trying to improve these tests. What would make sense from that matplotlib link

Reply:
The fancy idea would be to compare the results with a stored picture of the plot using `compare_images`. 

We don't have to do that, and I don't even know if it's a good idea. It's just something I have toyed with and decided to throw at you :)